services/thermal_mgr/thermal_mgr.h

```diff
@@ -5,7 +5,7 @@
 
 typedef enum {
   THERMAL_MGR_EVENT_MEASURE_TEMP_CMD,
-  
+  THERMAL_MGR_EVENT_TEMP_CHANGE
```

Comment:
Make this event name more descriptive, temperature change is too generic as this event is when the over temperature occurs